scripts/test_notification.py

```diff
@@ -228,7 +228,7 @@ def event_down_command(data=event_down_sample):
 
 def spam_command(count=10):
     commands = get_notification_commands()
-    commandsToRun = random.sample(commands, count) 
+    commandsToRun = random.sample(commands, min(count, len(commands)))
```

Comment:
As long as you're fixing this (nice catch, btw), `commandsToRun` should probably be generated with `random.choice()` in a loop so that this will work as expected for large values of `count`